src/views/menus/AccountMenu.tsx

```diff
@@ -87,6 +95,11 @@ export const AccountMenu = () => {
 
   const { appleAppStoreUrl, googlePlayStoreUrl } = useMobileAppUrl();
 
+  const showConfirmPendingDeposit =
+    walletType === WalletType.Keplr &&
+    MustBigNumber(usdcBalance).toNumber() > AMOUNT_RESERVED_FOR_GAS_USDC &&
+    MustBigNumber(usdcBalance).minus(AMOUNT_RESERVED_FOR_GAS_USDC).toFixed(2) !== '0.00';
```

Comment:
Why the `.toFixed(2)` here. Is it an issue of rounding?

`!MustBigNumber(usdcBalance).minus(AMOUNT_RESERVED_FOR_GAS_USDC).isZero())